main/src/ca/uwaterloo/flix/api/lsp/Visitor.scala

```diff
@@ -15,6 +15,7 @@
  */
 package ca.uwaterloo.flix.api.lsp
 
+import ca.uwaterloo.flix.language.ast.Name.QName
```

Comment:
Why this?

Reply:
Just a mistake. An import I forgot to remove